.github/workflows/zowe-cli-plugin.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
     - name: Update Dependencies
       id: npm-update
-      uses: zowe-actions/octorelease-script@master

```

Comment:
I believe this is supposed to be `zowe-action/octorelease/script@v1`

Reply:
This still needs to be updated.